Tests/Parser/JsonMapTest.php

```diff
@@ -92,10 +93,6 @@ public function testProcess()
         self::assertEquals(['user', 'tag'], $parser->getResults()['tags']->getPrimaryKey(true));
     }
 
-    /**
-     * @expectedException \Keboola\Juicer\Exception\UserException
-     * @expectedExceptionMessage Missing mapping for 'first' in config.
```

Comment:
Pouzitie anotacii je deprecated v tomto pripade:
![image](https://user-images.githubusercontent.com/19371734/102205742-e10de700-3ecb-11eb-8acd-235fb365fd11.png)

Preto som to rovno fixol v miestach, ktore som menil. Pri aktualizacii PHP a phpunit, to bude potrebne spravit vsade.